kaolin/datasets/shapenet.py

```diff
@@ -126,15 +96,12 @@ class ShapeNet_Meshes(data.Dataset):
         train (bool): return the training set else the test set
         split (float): amount of dataset that is training out of 1
```

Comment:
```
split (float): fraction of the dataset to be used for training (>=0 and <=1)
```

Reply:
This is actually not part of my PR, we intend to destroy this class to replace it by the one I've made.